react/Gallery.js

```diff
@@ -29,20 +49,28 @@ class Gallery extends Component {
       </div>
     )
   }
+
+  getItemsPerRow = () => {
+    const { maxItemsPerRow, gap, minItemWidth, width } = this.props
+    let maxItems = maxItemsPerRow
+    while ((minItemWidth + gap) * maxItems >= width) {
+      --maxItems
+    }
+    return maxItems
```

Comment:
Isn't `maxItems = width / (minItemWidth + gap)` ?

Reply:
Really is better :)